go/store/nbs/store.go

```diff
@@ -361,24 +361,17 @@ func fromManifestAppendixOptionNewContents(upstream manifestContents, appendixSp
 	}
 }
 
-// GetManifestStorageVersion returns the manifest storage version or an error if the operation
-// is not supported
+// GetManifestStorageVersion returns the manifest storage version or an error if the operation is not supported
 func (nbs *NomsBlockStore) GetManifestStorageVersion(ctx context.Context) (version string, err error) {
-	info, ok := nbs.mm.m.(ManifestInfo)
```

Comment:
@andrew-wm-arthur This was wrong... the `manifest` doesnt implement the `ManifestInfo` interface, `manifestContents` implements it... this PR fixes this mistake